### PR TITLE
Handle root network device names other than eth.

### PR DESCRIPTION
**Pull request by rjnagal**
New logic ignores veth, docker, and loopback devices.

### CODE REVIEW

**Comment by rjnagal** [comment]
Fixes #436 Verified on CoreOS.

File: utils/sysinfo/sysinfo.go [hunk 1]
```diff
@@ -77,8 +77,16 @@ func GetNetworkDevices(sysfs sysfs.SysFs) ([]info.NetInfo, error) {
 	netDevices := []info.NetInfo{}
 	for _, dev := range devs {
 		name := dev.Name()
-		// Only consider ethernet devices for now.
-		if !strings.HasPrefix(name, "eth") {
+		// Ignore docker, loopback, and veth devices.
+		ignoredDevices := []string{"lo", "veth", "docker"}
```
Comment (vmarmol) [inline utils/sysinfo/sysinfo.go]: nit: can we use a map instead :)

Comment (rjnagal) [inline utils/sysinfo/sysinfo.go]: I thought it won't help because we have to match just the prefix?

Comment (vmarmol) [inline utils/sysinfo/sysinfo.go]: Ah, good point.

**Comment by vmarmol** [comment]
LGTM

**Pull request merged**